pkg/component/kubeapiserver/kube_apiserver.go

```diff
@@ -196,6 +197,8 @@ type ServiceAccountConfig struct {
 	Issuer string
 	// AcceptedIssuers is an additional set of issuers that are used to determine which service account tokens are accepted.
 	AcceptedIssuers []string
+	// JWKSURI is used to overwrite the URI for the JSON Web Key Set in the discovery document served at /.well-known/openid-configuration.
+	JWKSURI *string
```

Comment:
A cosmetic suggestion
```suggestion
	jwksURI *string
```

Reply:
But then it cannot be accessed outside the package.

Reply:
Good question, please ignore my suggestion.